fc/jira/issue.py

```diff
@@ -23,6 +23,7 @@ class Issue:
     summary_jira_field = 'summary'
     description_jira_field = 'description'
     project_jira_field = 'project'
+    score_jira_field = 'customfield_18402'
```

Comment:
I do not believe this (and the other lines in this file) should be here.  `Issue` represents any issue across all of Jira.  QPPA wouldn't have (necessarily) a score.